src/Polynomials.jl

```diff
@@ -348,14 +348,24 @@ end
 div(num::Poly, den::Poly) = divrem(num, den)[1]
 rem(num::Poly, den::Poly) = divrem(num, den)[2]
 
-function ==(p1::Poly, p2::Poly)
-    if p1.var != p2.var
-        return false
-    else
-        return p1.a == p2.a
-    end
+==(p1::Poly, p2::Poly) = (p1.var == p2.var && p1.a == p2.a)
+==(p1::Poly, n::Number) = (coeffs(p1) == [n])
+==(n::Number, p1::Poly) = (p1 == n)
+
+function isapprox{T,S}(p1::Poly{T}, p2::Poly{S};
+  rtol::Real = sqrt(eps(promote_type(T,S))), atol::Real = 0)
```

Comment:
Do you want to do this within this PR or should we merge this in, and do that in another?